external-links.md

```diff
@@ -1,2 +1,22 @@
 ## External links
 
+#### Would you like to contribute to the OpenMRS Android client?
+
+See open Android client issues [here](https://issues.openmrs.org/projects/AC/issues). You will need an [OpenMRS ID](https://id.openmrs.org/) to be able to claim those issues.
+
+To start claiming issues on JIRA (where we manage our issues and projects), you will need to [create a new helpdesk case](https://help.openmrs.org/customer/portal/emails/new) to get permission to access JIRA.
+
+#### Setup your own server
+
+If you would like to set up a customized server, you may request one from helpdesk or server maintainers. Since resources are limited, these hosted developer test environments should be reserved for community collaboration projects and used for short periods of time (days or weeks).
+
+For more info, see this [wiki page](https://wiki.openmrs.org/x/-oO8Bg).
```

Comment:
Getting `Not Permitted` error on opening this link.
![screen shot 2018-01-10 at 8 06 48 pm](https://user-images.githubusercontent.com/9693420/34777903-c72ee882-f641-11e7-8770-5922e154089e.png)

Reply:
@shivtej1505, It works fine for me. It should take you to https://wiki.openmrs.org/display/ISM/OpenMRS+environments.
I had that error too, you will need to log out and log in again in Openmrs Wiki. Then it will work :D